api/app/controllers/transaction.controller.ts

```diff
@@ -60,8 +62,8 @@ exports.create = async (req, res) => {
     reviewId: req.body.reviewId ?? null,
     amount: +req.body.amount,
     transactionType: req.body.transactionType,
-    transactionStatus: TRANSACTION_STATUS.SUCCESS,
-    walletId: userWallet.id,
+    transactionStatus: TRANSACTION_STATUS.SUCCESS as TRANSACTION_TYPE_VALUES,
```

Comment:
Why must be do a type casting here? What error does typescript throw if do not do this? We should fix the underlying error or at least use a `satisfies` but not type cast

Reply:
`satisfies` wouldn't suffice, that's because the `transactionStatus` property in the object is mutated down the line in [line 87](https://github.com/bitcointranscripts/transcription-review-backend/pull/83/files/ca6845b22e8a2c01434dc006e80502545d814f51#diff-12377f5b32cb088f7157f57497850257ede154b67f1f927db4c4212c73e386efR87).

The other solution would be to create a type for the transaction object and ensure it conforms to that type. That way we wouldn't have to use `as` because of the mutation.